java-manta-client/src/main/java/com/joyent/manta/client/crypto/MantaEncryptedObjectInputStream.java

```diff
@@ -731,6 +746,12 @@ private void annotateException(final ExceptionContext exception) {
         exception.setContextValue("authenticationEnabled", this.authenticateCiphertext);
         exception.setContextValue("threadName", Thread.currentThread().getName());
 
+        final String requestId = MDC.get(RequestIdInterceptor.MDC_REQUEST_ID_STRING);
```

Comment:
This makes sense for `MantaObjectOutputStream` since there isn't a response directly accessible but it seems more straightforward to use `this.getRequestId()` here unless I'm missing something. All the constructors for `MantaObjectEncryptedInputStream` and `MantaObjectInputStream` seem to have access to a `MantaObjectResponse`

Either `getRequestId()` or `getHeaderAsString(MantaHttpHeaders.REQUEST_ID)` would be preferable I think.